src/services/AzureSubscriptionProvider.ts

```diff
@@ -184,7 +184,7 @@ export class VSCodeAzureSubscriptionProvider extends vscode.Disposable implement
                     authentication: {
                         getSession: () => session
                     },
-                    environment: {} as Environment,
+                    environment: Environment.AzureCloud,
```

Comment:
Can we hardcode this to `AzureCloud` or do we need to support other environments?

Reply:
I think we're only trying to support `AzureCloud` at the moment.

Reply:
Ok, that's fine. Are we planning to add support for the other clouds later?

Reply:
For the first iteration were not planning to support any other clouds, but as we migrate the vs code auth provider we may add support for sovereign/custom cloud.